enderio-machines/src/main/java/com/enderio/machines/common/blocks/farming_station/FarmingStationBlockEntity.java

```diff
@@ -98,11 +94,25 @@ public class FarmingStationBlockEntity extends PoweredMachineBlockEntity impleme
 
     private ActionRange actionRange = DEFAULT_RANGE;
 
+    private FakePlayer farmPlayer;
+
     public FarmingStationBlockEntity(BlockPos worldPosition, BlockState blockState) {
         super(MachineBlockEntities.FARMING_STATION.get(), worldPosition, blockState, true, CapacitorSupport.REQUIRED,
                 EnergyIOMode.Input, ENERGY_CAPACITY, ENERGY_USAGE);
     }
 
+    @Override
+    public void setLevel(Level level) {
+        super.setLevel(level);
+        if (level instanceof ServerLevel serverLevel) {
+            if (this.getMachineOwner() == null) {
+                this.setMachineOwner(UUID.randomUUID()); // Fallback
+            }
+            farmPlayer = FakePlayerFactory.get(serverLevel,
+                    new GameProfile(getMachineOwner(), "enderio:farm:" + worldPosition));
```

Comment:
Is it right that the fake player is given the machine owner's UUID?
Also - do we really need one fake player for every world position?

Reply:
This is what claims mods told me, make the fake players have the same uuid as the player (and you can have multiple with that uuid). As for if we need one for each position, for the farm we technically don't need it, but I thought it might be useful. We can remove it

Reply:
Yeah makes sense with the UUID, but yeah would rather there not be a fake player for every position